src/main/java/com/internetitem/logback/elasticsearch/writer/ElasticsearchWriter.java

```diff
@@ -52,7 +62,27 @@ public void sendData() throws IOException {
 			return;
 		}
 
-		HttpURLConnection urlConnection = (HttpURLConnection)(settings.getUrl().openConnection());
+		HttpURLConnection urlConnection;
+
+		URL url = settings.getUrl();
+
+		if ("https".equals(url.getProtocol().toLowerCase())){
+
+			trustAllHosts();
```

Comment:
I'm a bit uncomfortable with this and disabling hostname verification. If this can't be set at the JVM-level, perhaps this could be refactored out into something like a URLConnectionFactory or something (or put this stuff into a common method so the default implementation leaves the JVM defaults in place, but if you wanted to subclass to disable security, you could do it all in one place)